pages/in/tx/iscn/_id/index.vue

```diff
@@ -396,6 +396,25 @@ export default {
     totalStakeholdersShares() {
       return this.stakeholders.reduce((t, s) => t + s.rewardProportion, 0);
     },
+    creatorLikerId() {
+      if (this.creatorId.startsWith(EXTERNAL_URL)) {
+        this.creatorId.replace(EXTERNAL_URL, '');
```

Comment:
`String.prototype.replace` is not in-place, so this line has no effect

Reply:
Just got the idea, maybe you want `return this.creatorId.replace(EXTERNAL_URL, '');`?
The name `EXTERNAL_URL` is quite misleading in this context, since the logic is `if it starts with our "external URL", that means it's something internal and is a Liker ID`

Reply:
`EXTERNAL_URL` is computed from `EXTERNAL_HOSTNAME` of the site though..